### PR TITLE
Add subscribe feature

### PR DESCRIPTION
**Pull request by karl-jerico**
### OVERVIEW

- Add subscribe feature

### SNIPPETS

image

### CODE REVIEW

File: src/components/shop/newsLetter.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        <p>Subscribe to our newsletter and stay updated</p>
+        <div>
+            <input type="emai" placeholder='Your Email id' />
+            <button>Subscribe</button>
+        </div>
+    </div>
+  )
+}
+
+export default NewsLetter
```
Comment (mikco-hipe) [inline src/components/shop/newsLetter.jsx]: new line

File: src/css/components/shop/newsletter.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+.newsletter div button{
+    width: 210px;
+    height: 70px;
+    border-radius: 80px;
+    background: black;
+    color: white;
+    font-size: 16px;
+    cursor: pointer;
+}
```
Comment (mikco-hipe) [inline src/css/components/shop/newsletter.css]: new line

File: src/pages/shop.jsx [hunk 3]
```diff
@@ -3,6 +3,7 @@ import Hero from '@/components/shop/hero'
 import Popular from '@/components/shop/popular'
 import Offers from '@/components/shop/offers'
 import NewCollections from '@/components/shop/newCollections'
+import NewsLetter from '../components/shop/newsLetter'
```
Comment (mikco-hipe) [inline src/pages/shop.jsx]: use @

**Review by mikco-hipe — CHANGES_REQUESTED** [review]

**Review by karljerico — APPROVED** [review]

**Review by mikco-hipe — APPROVED** [review]
LGTM 🙇

**Review by Shiara-Deudor — APPROVED** [review]

**Pull request merged**